rfd/0001-general-design.md

```diff
@@ -0,0 +1,288 @@
+---
+authors: Anton Repin (github.com/xdire)
+state: wip
+---
+
+# RFD 1 - General Design
```

Comment:
Given that you've switched from a dynamic config API to static configuration, it'd be helpful to see an example of the structure of the static configuration whether that's the configuration file shape or the CLI flags that will be available when executing the load-balancer. I find that considering the configuration ahead of time often reveals complexities in the UX of operating a thing.